src/Database/Schema/TableSchemaInterface.php

```diff
@@ -79,6 +86,13 @@ interface TableSchemaInterface extends SchemaInterface
      */
     public const TYPE_TIMESTAMP_FRACTIONAL = 'timestampfractional';
 
+    /**
+     * Timestamp with time zone column type
+     *
+     * @var string
+     */
+    public const TYPE_TIMESTAMP_TIMEZONE = 'timestamptimezone';
```

Comment:
Given that MySQL and SQLite don't have a way of supporting this type should we fallback to a timezone-less storage column or raise an exception indicating that a fixture schema object cannot be converted into this dialect of SQL?

Reply:
Let me see if there's a consistent way to do that.

Reply:
This would require throwing an exception from BaseSchema::columnSql().  Nothing else checks or throws an error here.  The TestFixture wrapper will catch an exception (which it shouldn't) when the create sql fails to run and log it.

Reply:
Added default behavior for mysql, sqlserver and sqlite so fixtures can use these types without unexpected results.